src/libcrun/cgroup.h

```diff
@@ -65,8 +65,7 @@ LIBCRUN_PUBLIC int libcrun_move_process_to_cgroup (pid_t pid, pid_t init_pid, ch
 LIBCRUN_PUBLIC int libcrun_update_cgroup_resources (int cgroup_mode,
                                                     runtime_spec_schema_config_linux_resources *resources, char *path,
                                                     libcrun_error_t *err);
-LIBCRUN_PUBLIC int libcrun_cgroup_is_container_paused (const char *cgroup_path, int cgroup_mode, bool *paused,
-                                                       libcrun_error_t *err);
+LIBCRUN_PUBLIC int libcrun_cgroup_is_container_paused (const char *cgroup_path, bool *paused, libcrun_error_t *err);
```

Comment:
it is a breaking change in a public libcrun function, but I guess we probably don't hurt anyone :-) and the API is nicer

Reply:
Debian packaging might complain about it (I barely remember they actually check that).